src/overlays/actors/ovl_En_Tubo_Trap/z_en_tubo_trap.c

```diff
@@ -240,18 +245,18 @@ void func_80931004(EnTuboTrap* this, GlobalContext* globalCtx) {
         0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
     };
 
-    if ((this->actor.xzDistToPlayer < 200.0f) && (this->actor.world.pos.y <= player->base.world.pos.y)) {
+    if ((this->actor.xzDistToPlayer < 200.0f) && (this->actor.world.pos.y <= player->actor.world.pos.y)) {
         startingRotation = this->actor.home.rot.z;
         if ((startingRotation == 0) || (this->actor.yDistToPlayer <= ((f32)startingRotation * 10.0f))) {
             func_800BC154(globalCtx, &globalCtx->actorCtx, this, ACTORCAT_ENEMY);
             currentHeight = this->actor.world.pos.y;
             this->actor.flags |= 0x11; // always update and can target
 
-            // hard to know what this value is even used for without knowing what ActorPlayer::unk14B is
+            // hard to know what this value is even used for without knowing what Player::unk14B is
             // wild guess: this is player animation state, height is modified to always point at center of link model
             weirdvalue = D_8093146C[player->unk14B & 0xFF];
```

Comment:
This value, this becomes player->transformation

and the speculative comment above which was correct I guess

Reply:
Yeah, I'll just remove the comment and change it.